tests/Unit/PointwiseFunctions/AnalyticSolutions/WaveEquation/Test_PlaneWave.cpp

```diff
@@ -157,12 +157,22 @@ void test_1d() {
       std::array<DataVector, 2>{{-6.0 * omega * kx * u, 6.0 * square(kx) * u}},
       deserialized_pw, x, t);
 
-  test_creation<ScalarWave::Solutions::PlaneWave<1>>(
-      "  WaveVector: [3.5]\n"
-      "  Center: [3.5]\n"
-      "  Profile:\n"
-      "    PowX:\n"
-      "      Power: 4");
+  const auto created_solution =
+      test_creation<ScalarWave::Solutions::PlaneWave<1>>(
+          "  WaveVector: [3.5]\n"
+          "  Center: [3.5]\n"
```

Comment:
The only thing comparing these to explicitly constructed objects does is verifies that the factory and constructor agree on the order of the arguments, so make all the arguments different so that that is checked.

Reply:
Updated to use the same arguments as when constructing the solution object `pw` earlier in the test, which a) results in different arguments in the different slots, as requested, and b) simplifies the code by reusing `pw` in the comparison.